decidim-bulletin_board-app/cypress/integration/sandbox/election.spec.js

```diff
@@ -56,11 +56,11 @@ describe("Election", () => {
     });
   });
 
-  it("complete the whole process using the election_guard voting scheme", () => {
+  it("complete the whole process using the electionguard voting scheme", () => {
     testElection({
-      electionTitle: "My election_guard election",
-      electionUniqueId: "election_guard-1",
-      electionVotingSchemeName: "election_guard",
+      electionTitle: "My ElectionGuard election",
```

Comment:
It looks like this has been camel-cased on purpose, right?
Just wondering if it was a find&replace issue, that might have had an impact in more critical spots (haven't spotted any other inconsistency, but I'm only human, after all 🤖 ).

Reply:
Mmm, you are right that this is not consistent with the rest of the changes, but it shouldn't affect anything in this case, as this is just a regular string. 

The main problem is that Microsoft uses `ElectionGuard` for the title of the project but `electionguard` for naming all identifiers (projects, namespaces, etc). So, I'm trying to keep that naming schema, but Rails will search for `election_guard` when you define an `ElectionGuard` class. That was why it was done as it was. Then, when mixing Ruby and Python code, it was very hard to keep it consistent. I think that the less worst solution is to keep using  `electionguard` everywhere, and use `Electionguard` for class names on Ruby.